docs/src/pages/guides/FHIRSearchConfiguration.md

```diff
@@ -18,14 +18,14 @@ The IBM FHIR Server allows deployers to define search parameters on a tenant-spe
 Tenant search parameters are defined via a [Bundle](https://www.hl7.org/fhir/r4/bundle.html) of [SearchParameter](https://www.hl7.org/fhir/r4/searchparameter.html) resources that define the additional search parameters which describe the searchable field and define the FHIRPath expression for extraction.  For example, a tenant that extends the `Patient` resource type with the `favorite-color` extension, enables search on `favorite-color` by defining a SearchParameter as part of this bundle.
 
 ## 1 Configuration
-There are three layers of search parameter configuration.  
-- built-in parameters from the core specification and packaged implementation guides
-- default tenant parameters
-- tenant-specific parameters
+Search parameter definitions come from the core specification, packaged implementation guides, and other resources provided to the registry via the FHIRRegistryResourceProvider SPI.
```

Comment:
```suggestion
Search parameter definitions come from the core specification, packaged implementation guides, and  resources provided to the IBM FHIR Server's registry (FHIRRegistry) via the FHIRRegistryResourceProvider SPI.
```
I had to read the above twice for clarity, when I reread without "other", the sentence was a bit easier to read.  I'm not sure that we mention the registry elsewhere in this document, so I clarified it's the IBM FHIR Server registry.

Reply:
Given that the very next paragraph is all about loading the SearchParameter definitions from the registry, I think I might be able to nix the whole sentence.

Reply:
done